scout/server/templates/layout.html

```diff
@@ -50,6 +50,17 @@
           <ul class="navbar-nav navbar-right">
             {% block top_nav_right %}
               {% if current_user.is_authenticated %}
+                <li class="nav-item dropdown">
+                  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown">Timeline<span class="caret"></span></a>
+                  <ul class="dropdown-menu">
+                    <li>
+                      <a class="dropdown-item" href="{{ url_for('overview.timeline', limit=100) }}">Last events</a>
+                    </li>
+                    <li>
+                      <a class="dropdown-item" href="{{ url_for('overview.timeline', limit=-1) }}">All events</a>
```

Comment:
Heh, that is a very long list for some users. One could paginate, filter on time, or allow users to set a count. But, as always, improvements can happen later - having a start is good.

Reply:
In the lastest version of this PR I've included this: 
<img width="210" alt="image" src="https://user-images.githubusercontent.com/28093618/174532322-4d22e10c-93a6-4b8c-8979-3cb903f703c6.png">

I've tested on stage with my user and it's fast enough to retrieve all events for my user since 2018. I'm wondering how it looks like for you (you should have many more events in the database than me). If it gets stuck I can change it to show less events instead!

Reply:
Yes, again, good start - that was actually there, I updated stage with what was on the repo when I reviewed. It loads, and rather quickly - then our highest ranking officers can have an order of magnitude more. I dont know what the best is, but on the order of hundreds sounds like a starting point. Perhaps make the lowest level like a few hundred instead? The goal should be to cover what one might keep open in tabs for later perusal over a few days.

Reply:
I've changed it to display 50 and 500 events now:

<img width="139" alt="image" src="https://user-images.githubusercontent.com/28093618/174544410-bb90b27b-bea9-418d-ae32-520d565d7f92.png">


The upper level should give just the latest stuff. 500 events should go back in time a bit more (a month or so perhaps) for the most active users. Does it look OK?